django_mongodb/features.py

```diff
@@ -344,4 +356,17 @@ class DatabaseFeatures(BaseDatabaseFeatures):
             "db_functions.comparison.test_cast.CastTests.test_cast_from_python_to_datetime",
             "db_functions.comparison.test_cast.CastTests.test_cast_to_duration",
         },
+        "Mongodb support uuid as a type in json fields.": {
```

Comment:
"MongoDB supports UUIDs in JSONField." (But actually, this test might work if `adapt_json_value()` uses `encoder`.)